pkg/ccl/changefeedccl/kvevent/event.go

```diff
@@ -23,7 +23,22 @@ import (
 
 // ErrBufferClosed is returned by Readers when no more values will be
 // returned from the buffer.
-var ErrBufferClosed = errors.New("buffer closed")
+type ErrBufferClosed struct {
+	reason error
+}
+
+// Error() implements the error interface
+func (e ErrBufferClosed) Error() string {
+	return "buffer closed"
```

Comment:
Should we include the message from the reason here?

Reply:
This string is only being used as a sentinel error, it's the reason that'll end up being displayed.